Jest not able to find test specs in Windows 10

Fresh project with 'vue init webpack just-testing' doesn't find the test files in Win 10
...
  25 files checked.
  testMatch: **/__tests__/**/*.js?(x),**/?(*.)(spec|test).js?(x) - 1 match
  testPathIgnorePatterns: \\node_modules\\ - 25 matches
Pattern: test/unit/specs - 0 matches

@eddyerburgh I doubt it will since that isn't where my test specs are
@Akryum See below:

```
 FAIL  test\e2e\specs\test.js
  ● Test suite failed to run

    Your test suite must contain at least one test.

      at node_modules/jest/node_modules/jest-cli/build/test_scheduler.js:95:22
          at Generator.next (<anonymous>)
          at Promise (<anonymous>)
          at <anonymous>
      at process._tickCallback (internal/process/next_tick.js:188:7)

 PASS  test\unit\specs\HelloWorld.spec.js

Test Suites: 1 failed, 1 passed, 2 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        10.432s
Ran all test suites.
-----------------|----------|----------|----------|----------|----------------|
File             |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
-----------------|----------|----------|----------|----------|----------------|
All files        |     7.69 |        0 |    33.33 |     12.5 |                |
 src             |        0 |        0 |        0 |        0 |                |
  App.vue        |        0 |        0 |        0 |        0 |... 19,20,21,22 |
 src/components  |      100 |      100 |      100 |      100 |                |
  HelloWorld.vue |      100 |      100 |      100 |      100 |                |
-----------------|----------|----------|----------|----------|----------------|
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! test-jest@1.0.0 unit: `jest --config test/unit/jest.conf.js --coverage`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the test-jest@1.0.0 unit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\robert\AppData\Roaming\npm-cache\_logs\2017-11-16T15_08_14_710Z-debug.log
```

--- a/meta.js
+++ b/meta.js
@@ -107,6 +107,7 @@ module.exports = {
     "config/test.env.js": "unit || e2e",
     "test/unit/**/*": "unit",
     "test/unit/index.js": "unit && runner === 'karma'",
+    "test/unit/jest.conf.js": "unit && runner === 'jest'",
     "test/unit/karma.conf.js": "unit && runner === 'karma'",
     "test/unit/specs/index.js": "unit && runner === 'karma'",
     "test/unit/setup.js": "unit && runner === 'jest'",

--- a/template/package.json
+++ b/template/package.json
@@ -113,33 +113,6 @@
     "webpack-dev-server": "^2.9.1",
     "webpack-merge": "^4.1.0"
   },
-  {{#if_eq runner "jest"}}
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "json",
-      "vue"
-    ],
-    "moduleNameMapper": {
-      "^@/(.*)$": "<rootDir>/src/$1"
-    },
-    "transform": {
-      "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
-      ".*\\.(vue)$": "<rootDir>/node_modules/vue-jest"
-    },
-    "setupFiles": ["<rootDir>/test/unit/setup"],
-    "mapCoverage": true,
-    "coverageDirectory": "<rootDir>/test/unit/coverage",
-    "collectCoverageFrom" : [
-      "src/**/*.{js,vue}",
-      "!src/main.js",
-      {{#router}}
-      "!src/router/index.js",
-      {{/router}}
-      "!**/node_modules/**"
-    ]
-  },
-  {{/if_eq}}
   "engines": {
     "node": ">= 4.0.0",
     "npm": ">= 3.0.0"

--- a/template/package.json
+++ b/template/package.json
@@ -8,7 +8,7 @@
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
     "start": "npm run dev",
     {{#if_eq runner "jest"}}
-    "unit": "jest test/unit/specs --coverage",
+    "unit": "jest --config test/unit/jest.conf.js --coverage",
     {{/if_eq}}
     {{#if_eq runner "karma"}}
     "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
@@ -68,6 +68,7 @@
     {{#if_eq runner "jest"}}
     "babel-jest": "^21.0.2",
     "jest": "^21.2.0",
+    "jest-serializer-vue": "^0.3.0",
     "vue-jest": "^1.0.2",
     {{/if_eq}}
     {{#if_eq runner "karma"}}

--- a/template/test/unit/jest.conf.js
+++ b/template/test/unit/jest.conf.js
@@ -1,0 +1,29 @@
+const path = require('path')
+
+module.exports = {
+  rootDir: path.resolve(__dirname, '../../'),
+  moduleFileExtensions: [
+    'js',
+    'json',
+    'vue'
+  ],
+  moduleNameMapper: {
+    '^@/(.*)$': '<rootDir>/src/$1'
+  },
+  transform: {
+    '^.+\\.js$': '<rootDir>/node_modules/babel-jest',
+    '.*\\.(vue)$': '<rootDir>/node_modules/vue-jest'
+  },
+  snapshotSerializers: ['<rootDir>/node_modules/jest-serializer-vue'],
+  setupFiles: ['<rootDir>/test/unit/setup'],
+  mapCoverage: true,
+  coverageDirectory: '<rootDir>/test/unit/coverage',
+  collectCoverageFrom: [
+    'src/**/*.{js,vue}',
+    '!src/main.js',
+    '{{#router}}
+    "!src/router/index.js",
+    {{/router}}
+    '!**/node_modules/**'
+  ]
+}

--- a/template/test/unit/jest.conf.js
+++ b/template/test/unit/jest.conf.js
@@ -21,8 +21,8 @@ module.exports = {
   collectCoverageFrom: [
     'src/**/*.{js,vue}',
     '!src/main.js',
-    '{{#router}}
-    "!src/router/index.js",
+    {{#router}}
+    '!src/router/index.js',
     {{/router}}
     '!**/node_modules/**'
   ]

